tests/dynamic_data/publications/crs/vectors/precision_test.py

```diff
@@ -205,6 +206,7 @@ class StyleFileDomain(base_test.StyleFileDomainBase):
 }
 
 
+@pytest.mark.xfail(reason='Geoserver proxy is not yet ready for WFS layers are by UUID so they do not yet refresh after feature change', strict=False)
```

Comment:
"do not yet refresh after feature change" does not make sense to me in this test

Reply:
Incorporated.